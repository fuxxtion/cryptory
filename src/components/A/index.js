import React from 'react'

export default props => <a href={props.link}>{props.text}</a>