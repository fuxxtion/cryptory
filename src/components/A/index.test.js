import React from 'react'
import A from './index'
import renderer from 'react-test-renderer'

test('Initial render', () => {
  const component = renderer.create(
    <A link="javascript:void(0)" text="test" />
  )

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot();
})