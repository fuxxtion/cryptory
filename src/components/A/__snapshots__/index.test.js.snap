// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Initial render 1`] = `
<a
  href="javascript:void(0)"
>
  test
</a>
`;
